### ISSUE TITLE
Is there a configuration for tag names?

### ISSUE THREAD

**Issue by chadmalla-ff** [springdoc/springdoc-openapi#173]
For my project I just want to set the tag name to the name of the controller class. Is there a way where I can configure this in the `OpenAPI` bean so that I don't have to annotate my controller with `@Tag(name = "", description = "")` for each controller?

Thanks!

**Comment by springdoc** [comment]
Hi, 
 
 
  You can add a custom implementation of RequestBuilder Bean at your application level, and then override customiseOperation.
 For example:
```java
@Override
protected Operation customiseOperation(Operation operation, HandlerMethod handlerMethod) {
	operation.addTagsItem(handlerMethod.getBeanType().getSimpleName());
	return operation;
}
```

**Comment by springdoc** [comment]
Another PR proposed by @hisener  will be available on next release:1.2.10
